### DISCUSSION TITLE
样式修改

### DISCUSSION THREAD

**Discussion by zenggpzqbx** [element-plus/element-plus#11981] [Feature Request]
Feature type: **`Other`**

## Details

- Enhanced component name: `All`

## Description

给每一个组件定义一个class或id, 通过class或id可以直接修改样式：el-radio-button按钮点击后的样式.

### Additional comments

现在修改样式的方法不够便捷，有很多的情况是：网上搜了一大堆资料，没一个是起效的，影响开发效率

**Comment by gimjin** [comment]
推荐用unocss。 [请看 demo](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwCIYAsxxAJwFMA7ALm1wJPIHs4b8BaIkdAPgCgoo5CJC4UAIxcooSJ27de-QQCZR3cRElTpfAVADMisQHpwK0cAMSOJw2yA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwod6gBccADawRrCbFQwagC0fraoGmRIorbQQRqUyBqKALq5JN5wVAAW0DARMFRBXj4%2B4ZHRIDGl5ZUaGTCmMDG2RHBdJj3QVlhEqHBlAG6oUAGDPV1WtklFCoU%2B-FLVIUWbkrSyEO4AlBt1W7QMTKwcXNyn5-sCMCJinnt1VMLLqEEAjAA6ABMTzqwmQWAyJUYYGqAHp4dQQIDSvDVnVFGcigUDAoTgYgA&css=Q&options=N4IgzgLgTglgxhEAuaBXApgGhNAhgOzADMB7KAW2SNwBswsQp0wAHEwmAN3StvoF8gA)
